drivers/gles3/rasterizer_storage_gles3.h

```diff
@@ -302,7 +302,7 @@ inline void RasterizerStorageGLES3::buffer_orphan_and_upload(unsigned int p_buff
 }
 
 inline String RasterizerStorageGLES3::get_framebuffer_error(GLenum p_status) {
-#ifdef DEBUG_ENABLED
+#if defined(DEBUG_ENABLED) && defined(GLES_OVER_GL)
```

Comment:
For future reference only ``GL_FRAMEBUFFER_INCOMPLETE_DRAW_BUFFER`` and ``GL_FRAMEBUFFER_INCOMPLETE_READ_BUFFER`` are only available on desktop GL. The first two are mandated even in GLES 2.0. That being said, these strings are most helpful when debugging from desktop. so this change is fine